File: python/worker-payments/payments.py
def payment(worker_type, hours):
    rates = {
        's4': 1500,
        's6': 25000,
        'univ': 35000
    }
    
    if worker_type in rates:
        return rates[worker_type] * hours
    else:
        raise ValueError("Invalid worker type")


def payment2(worker_type, hours):
    try:
        rate = {'s4': 15000, 's6': 25000, 'univ': 35000}[worker_type]
    except KeyError:
        raise ValueError("Invalid worker type")
    return rate * hours

File: javascript/worker-payments/payments.js
function payment(level, hours) {
    const rates = {
        s4: 15000,
        s6: 25000,
        univ: 35000
    }
    if (level === 's4') {
        return hours*rates['s4']
    }

    if (level === 's6') {
        return hours*rates['s6']
    }

    if (level === 'univ') {
        return hours*rates['univ']
    }
}

module.exports  = payment
